lib/extensions/params.py

```diff
@@ -525,7 +525,7 @@ def tab_sort_key(tab):
 
                 # If multiple tabs are enabled, make sure dependent
                 # tabs are grouped with the parent.
-                parent,
+                parent and parent.name,
```

Comment:
Hi!  Thanks for your PR, we really appreciate your contribution.

What is this change about?  Use it not working as just `parent`?

Reply:
Yes, this fixes the error mentioned in https://github.com/inkstitch/inkstitch/issues/980#issuecomment-762686051